Build error in gcc 10.1.1

You get `multiple definitionof 'ngx_php_request'`, `multiple definition of 'old_zend_error_cb'` error when building on fedora 32.

- os: fedora 32
- gcc: 10.1.1
- nginx: 1.18.0
- php: 7.4.6

```
...
cc -o objs/ngx_http_php_module.so \
objs/addon/src/ngx_http_php_module.o \
objs/addon/src/ngx_http_php_core.o \
objs/addon/src/ngx_http_php_directive.o \
objs/addon/src/ngx_http_php_handler.o \
objs/addon/src/ngx_http_php_request.o \
objs/addon/src/ngx_http_php_zend_uthread.o \
objs/addon/src/ngx_http_php_sleep.o \
objs/addon/src/ngx_http_php_socket.o \
objs/addon/src/ngx_http_php_util.o \
objs/addon/src/ngx_http_php_variable.o \
objs/addon/src/ngx_http_php_header.o \
objs/addon/src/ngx_http_php_keepalive.o \
objs/addon/src/ngx_php_debug.o \
objs/addon/src/ngx_php_conf_file.o \
objs/addon/impl/php_ngx.o \
objs/addon/impl/php_ngx_core.o \
objs/addon/impl/php_ngx_log.o \
objs/addon/impl/php_ngx_request.o \
objs/addon/impl/php_ngx_socket.o \
objs/addon/impl/php_ngx_var.o \
objs/addon/impl/php_ngx_sockets.o \
objs/addon/impl/php_ngx_header.o \
objs/addon/impl/php_ngx_cookie.o \
objs/ngx_http_php_module_modules.o \
-Wl,-z,relro -Wl,--as-needed -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -Wl,-E -Wl,-rpath,/usr/lib64 -lcrypt -lresolv -lcrypt -lncurses -lrt -lstdc++ -lrt -lm -ldl -lxml2 -lgssapi_krb5 -lkrb5 -lk5crypto -lcom_err -lssl -lcrypto -lpcre2-8 -lz -ledit -lcrypt -lcrypt -L/usr/lib64 -lphp7 -shared
/usr/bin/ld: objs/addon/src/ngx_http_php_core.o:(.bss+0x0): multiple definitionof `ngx_php_request'; objs/addon/src/ngx_http_php_module.o:(.bss+0x0): first defined here
/usr/bin/ld: objs/addon/src/ngx_http_php_core.o:/app/build/nginx-1.18.0/../../src/ngx_http_php_core.h:177: multiple definition of `old_zend_error_cb'; objs/addon/src/ngx_http_php_module.o:/app/build/nginx-1.18.0/../../src/ngx_http_php_core.h:177: first defined here
..
```

--- a/src/ngx_http_php_core.h
+++ b/src/ngx_http_php_core.h
@@ -174,7 +174,7 @@ int ngx_http_php_code_read_post(char *buffer, uint count_bytes TSRMLS_DC);
 char *ngx_http_php_code_read_cookies(TSRMLS_D);
 int ngx_http_php_code_header_handler(sapi_header_struct *sapi_header, sapi_header_op_enum op, sapi_headers_struct *sapi_headers TSRMLS_DC);
 
-void (*old_zend_error_cb)(int, const char *, const uint, const char *, va_list);
+extern void (*old_zend_error_cb)(int, const char *, const uint, const char *, va_list);
 void ngx_php_error_cb(int type, const char *error_filename, const uint error_lineno, const char *format, va_list args);
 
 #endif

--- a/src/ngx_http_php_module.c
+++ b/src/ngx_http_php_module.c
@@ -419,6 +419,10 @@ ngx_module_t ngx_http_php_module = {
     NGX_MODULE_V1_PADDING
 };
 
+ngx_http_request_t *ngx_php_request;
+
+void (*old_zend_error_cb)(int, const char *, const uint, const char *, va_list);
+
 static ngx_int_t 
 ngx_http_php_init(ngx_conf_t *cf)
 {

--- a/src/ngx_http_php_module.h
+++ b/src/ngx_http_php_module.h
@@ -50,7 +50,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #endif
 
 extern ngx_module_t ngx_http_php_module;
-ngx_http_request_t *ngx_php_request;
+extern ngx_http_request_t *ngx_php_request;
 
 typedef struct ngx_http_php_main_conf_s {
 

